packages/aragon-api/src/index.js

```diff
@@ -231,16 +274,49 @@ export class AppProxy {
         Promise.resolve(reducer(state, event))
```

Comment:
I think with RxJS 6 we can now remove the `Promise.resolve()` and just use `from()` directly :smile:

Reply:
As discussed. This is to ensure `from` doesn't mistakenly return an observable with multiple emissions in cases where a reducer returns an array.